app/src/main/res/layout-land/topic_practice_subtopic.xml

```diff
@@ -16,8 +16,9 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingStart="48dp"
-    android:paddingEnd="48dp"
+    android:layout_marginBottom="16dp"
+    android:layout_marginRight="96dp"
```

Comment:
Recommended to use margin start and margin end.

Reply:
Done